src/main/java/com/uid2/admin/vertx/service/SearchService.java

```diff
@@ -0,0 +1,87 @@
+package com.uid2.admin.vertx.service;
+
+import com.uid2.admin.auth.AdminUserProvider;
+import com.uid2.admin.vertx.ResponseUtil;
+import com.uid2.shared.auth.Role;
+import com.uid2.shared.auth.RotatingOperatorKeyProvider;
+import com.uid2.shared.middleware.AuthMiddleware;
+import com.uid2.shared.store.reader.RotatingClientKeyProvider;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.RoutingContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.vertx.core.http.HttpHeaders;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+public class SearchService implements IService {
+    private static final String queryParameter = "keyOrSecret";
```

Comment:
Better to rename this as `QUERY_PARAMETER` since it's a `static final` variable.

You could format these variables as such:
```
    private static final Logger LOGGER = LoggerFactory.getLogger(SearchService.class);
    private static final String QUERY_PARAMETER = "keyOrSecret";

    private final AuthMiddleware auth;
    private final RotatingClientKeyProvider clientKeyProvider;
    private final RotatingOperatorKeyProvider operatorKeyProvider;
    private final AdminUserProvider adminUserProvider;
```

Reply:
Done